### ISSUE TITLE
option for more contagious tropical disease

### ISSUE THREAD

**Issue by drzel** [FortressOne/server-qwprogs#689]
[23:25] Wolv: Should be that if one person gets it in the team, everyone does
[23:25] zel: It's already contagious... guess we could make it more contagious
[23:26] Wolv: What's the rate that it kills someone?
[23:26] Wolv: 5-6 hp per second or something
[23:27] zel: dunno
[23:28] zel: I think you have to touch a team mate, should make it so  it's just go within a certain range
[23:30] Wolv: It's not that easily spread atm.
[23:30] Wolv: I knows its touch, but it doesn't seem to spread when they touch

**Comment by Rickson-FOLD** [comment]
I agree that the disease rarely spreads. And it'd be nice if it did. Maybe make it so that any time a COVID-19 patient touches anyone, that person gets the infection. Only the infecting medic should be immune.

I think also that, perhaps only on the team with the COVID-19 patient, if anyone looks directly at the patient (the crosshair passes across), that person becomes infected. It's not terribly realistic, but I think it'd be an elegant way to code it in, and it sort of accounts for the fact that it's an airborne contagion.

**Comment by Galmeno** [comment]
Possibility to infect own spy, then let him disguise. Makes for an epic teamplay combo.
